### PR TITLE
[WIP] Add GitLab Runner package.

### PR DESCRIPTION
**Pull request by ssk2**

### CODE REVIEW

**Comment by ryadav88** [comment]
@tobilg can you update this PR to add your changes for Gitlab Runner?

**Comment by tobilg** [comment]
I created another PR for merging my changes into the `gitlab-ci` branch, see https://github.com/mesosphere/universe/pull/843

**Comment by ryadav88** [comment]
@tobilg ready to merge once we have an `example` in https://github.com/dcos/examples

File: repo/packages/G/gitlab-runner/0/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://secure.gravatar.com/avatar/6edd0acaf80f784fab3dd2c31d604e74.jpg?s=40&r=g&d=mm",
+    "icon-medium": "https://secure.gravatar.com/avatar/6edd0acaf80f784fab3dd2c31d604e74.jpg?s=80&r=g&d=mm",
+    "icon-large": "https://secure.gravatar.com/avatar/6edd0acaf80f784fab3dd2c31d604e74.jpg?s=200&r=g&d=mm"
```
Comment (ryadav88) [inline repo/packages/G/gitlab-runner/0/resource.json]: use the `gitlab runner` logo instead: https://gitlab.com/uploads/project/avatar/250833/runner_logo.png

File: repo/packages/G/gitlab-runner/0/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "gitlab-runner",
+  "version": "1.0-1.7.1",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://github.com/mesosphere/dcos-gitlab-runner-service.git",
+  "maintainer": "support@gitlab.com",
+  "website": "https://about.gitlab.com/gitlab-ci/",
+  "framework": false,
+  "description": "GitLab Runner is the open-source continuous integration (CI) service included with GitLab that coordinates the testing.",
```
Comment (ryadav88) [inline repo/packages/G/gitlab-runner/0/package.json]: add an example in dcos/examples and link it at the end of `description`

Comment (tobilg) [inline repo/packages/G/gitlab-runner/0/package.json]: Will do.

**Comment by joerg84** [comment]
@ssk2 is this still WIP?

**Comment by ssk2** [comment]
@joerg84: yes, that is why we haven't merged it. Awaiting feedback from GitLab's engineers.

**Comment by markpundsack** [comment]
@ssk2 Sorry, I'm just jumping in here. After the feature freeze for our next release, a GitLab engineer will review and provide feedback.

**Comment by tobilg** [comment]
Hi @markpundsack, thanks for the heads-up! When will be the feature freeze?

**Comment by markpundsack** [comment]
@tobilg Feature freeze is today. :)

**Comment by tmaczukin** [comment]
@ssk2 @tobilg I've got back to test this, but since todays morning I have problems with setting up DC/OS cluster. When I'll resolve this problem I'll test the integration again and leave a feedback for you.

**Comment by ryadav88** [comment]
@tobilg @ssk2 any update here?

**Comment by tobilg** [comment]
Waiting for the https://github.com/mesosphere/universe/pull/1126 to be merged, then testing with the Runner version 9.1

**Comment by ayufan** [comment]
OK. So we are currently waiting for https://github.com/mesosphere/universe/pull/1126?

**Pull request closed without merging**